<launch>
<node pkg="drone_project" name="testOpticalFlow" type="testOpticalFlow.py">
    <remap from="/image_in" to="/bebop/image_raw"/>
</node>
</launch>